addons/account_edi/models/account_edi_document.py

```diff
@@ -256,7 +256,7 @@ def _cron_process_documents_web_services(self, job_count=None):
 
         :param job_count: Limit explicitely the number of web service calls. If not provided, process all.
         '''
-        edi_documents = self.search([('state', 'in', ('to_send', 'to_cancel'))])
+        edi_documents = self.search([('state', 'in', ('to_send', 'to_cancel')), ('move_id.state', '!=', 'draft')])
```

Comment:
```suggestion
        edi_documents = self.search([('state', 'in', ('to_send', 'to_cancel')), ('move_id.state', '=', 'poŝted')])
```
At least it is a fail-safe.